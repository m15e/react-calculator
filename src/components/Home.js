import React from 'react';
import Header from './Header';

const Home = () => (
  <>
    <Header />
    <h2>Home here</h2>
  </>
);

export default Home;
