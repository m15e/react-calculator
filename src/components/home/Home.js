import React from 'react';
import Header from '../header/Header';

const Home = () => (
  <>
    <Header />
    <div className="content-container">
      <h2>Welcome to our Math Magicians!</h2>
      <p>
        A calculator built with React! Click the &quot;Calculator&quot; link above to check it out.
      </p>
    </div>
  </>
);

export default Home;
