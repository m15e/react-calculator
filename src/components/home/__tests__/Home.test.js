import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('Home component matches snapshot', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
