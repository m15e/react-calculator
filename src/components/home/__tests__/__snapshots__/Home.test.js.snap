// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches snapshot 1`] = `
Array [
  <div
    className="header-container"
  >
    <ul>
      <li>
        <a
          href="/"
        >
          Math Magicians
        </a>
      </li>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </div>,
  <div
    className="content-container"
  >
    <h2>
      Welcome to our Math Magicians!
    </h2>
    <p>
      A calculator built with React! Click the "Calculator" link above to check it out.
    </p>
  </div>,
]
`;
